src/panels/my/ha-panel-my.ts

```diff
@@ -51,6 +51,9 @@ export const getMyRedirects = (hasSupervisor: boolean): Redirects => ({
     component: "calendar",
     redirect: "/calendar",
   },
+  companion_app: {
+    redirect: "#external-app-configuration",
+  },
```

Comment:
```suggestion
  companion_app: {
    component: "mobile_app",
    redirect: "#external-app-configuration",
  },
```
Or can this work without that integration?

Reply:
Yes, it can work without.

See a little lower, the `this.hass.auth.external?.config.hasSettingsScreen` condition.

![CleanShot 2023-07-24 at 15 31 28](https://github.com/home-assistant/frontend/assets/195327/529cdb67-c2a9-42db-a695-e925ec98e127)

This is what is looks like from non-mobile / instance without mobile app.

Reply:
What I mean, can you visit this page from within the app, but without loading `mobile_app` integration.

Reply:
Can the mobile app, register things without that integration being present?

Reply:
Sorry, I wasn't clear: Yes :)